packages/components/src/draggable/index.native.js

```diff
@@ -121,6 +121,9 @@ const Draggable = ( { children, onDragEnd, onDragOver, onDragStart } ) => {
 				}
 			}
 		} )
+		.onTouchesCancelled( ( _event, state ) => {
+			state.end();
+		} )
```

Comment:
Awesome workaround for addressing the issue 🏅 , way better than listening to the `AppState` 😅 .

Reply:
Indeed! We have already too many listeners so I was happy when this approach worked for us 🎉